src/CreativeCommons/icons/CCIconsProps.js

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Necessary?

Reply:
This was an artefact from a failed build. I removed it.